internal/resources/managementclusterregistration/resource_management_cluster_registration.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package managementclusterregistration
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	clustermodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/cluster"
+	managementclusterregistrationmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/managementclusterregistration"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/common"
+)
+
+func ResourceManagementClusterRegistration() *schema.Resource {
+	return &schema.Resource{
+		Schema: managementClusterRegistrationSchema,
+		//CreateContext: , TODO
+		//ReadContext: , TODO
```

Comment:
I assumed this will be addressed in the resource implementation PR?

Reply:
Yes